Geometry/CaloTopology/src/HcalTopology.cc

```diff
@@ -571,8 +567,7 @@ bool HcalTopology::validRaw(const HcalDetId& id) const {
 
   if (ok) {
     if (subdet == HcalBarrel) {
-      if ((mode_ == HcalTopologyMode::SLHC) || (mode_ == HcalTopologyMode::H2HE) || (mode_ == HcalTopologyMode::Run3) ||
-          (mode_ == HcalTopologyMode::Run4)) {
+      if (phase2() || (mode_ == HcalTopologyMode::H2HE)) {
```

Comment:
This does not include Run3 now: is it correct? (Just checking)

Reply:
Yes. Earlier logic was wrong